docs/customization/menu.rst

```diff
@@ -41,32 +40,84 @@ In the example below we are adding a one new item to the Admin panel menu and a
         {
             $menu = $event->getMenu();
 
-            $menu->addChild('backend_main')
-                ->setLabel('Test Backend Main');
+            $newSubmenu = $menu->addChild('new')
+                ->setLabel('New Admin Panel Menu');
```

Comment:
when I saw the screenshot, I thought you added a functionality to build menus via UI in admin...